instrumentation/instaawssdk/invoke_lambda.go

```diff
@@ -0,0 +1,47 @@
+package instaawssdk
+
+import (
+	"github.com/aws/aws-sdk-go/service/lambda"
+	otlog "github.com/opentracing/opentracing-go/log"
+
+	"github.com/aws/aws-sdk-go/aws/request"
+	instana "github.com/instana/go-sensor"
+	"github.com/opentracing/opentracing-go"
+)
+
+func StartInvokeLambdaSpan(req *request.Request, sensor *instana.Sensor) {
+	parent, ok := instana.SpanFromContext(req.Context())
+	if !ok {
+		return
+	}
+
+	sp := sensor.Tracer().StartSpan("aws.lambda.invoke",
+		opentracing.ChildOf(parent.Context()),
+	)
+
+	if ii, ok := req.Params.(*lambda.InvokeInput); ok {
+		if ii.FunctionName != nil {
+			sp.SetTag("function", *ii.FunctionName)
+		}
+
+		if ii.InvocationType != nil {
+			sp.SetTag("type", *ii.InvocationType)
+		}
+	}
+
+	req.SetContext(instana.ContextWithSpan(req.Context(), sp))
+	injectTraceContext(sp, req)
+}
+
```

Comment:
Let's add a godoc here as well since this is an exported function

Reply:
It will be in upcoming commit. 👍